raco/backends/myria/myria.py

```diff
@@ -920,10 +920,12 @@ class MyriaQueryScan(algebra.ZeroaryOperator, MyriaOperator):
 
     """A Myria Query Scan"""
 
-    def __init__(self, sql, scheme, num_tuples=algebra.DEFAULT_CARDINALITY,
+    def __init__(self, sql, scheme, source_relation_keys,
+                 num_tuples=algebra.DEFAULT_CARDINALITY,
```

Comment:
Do we also need to change its `__repr__()`? (My understanding is that some tests may fail if `__repr__()` is inconsistent with the constructor..?)